### PR TITLE
Dependency fix

### PR DESCRIPTION
**Pull request by petschki**
Dependency on `ZEO` and `zdaemon` are lost when defining additional `eggs` in the recipe. Make sure we do not loose our required dependencies in that case.

see discussion https://community.plone.org/t/plone-6-0-4-released/17406/7?u=petschki

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
pre-commit.ci autofix

**Comment by petschki** [comment]
I think this issue gets fixed with this too https://github.com/plone/plone.recipe.zeoserver/issues/29

**Comment by petschki** [comment]
pre-commit.ci autofix

**Review by jensens — APPROVED** [review]

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: setup.py [hunk 1]
```diff
@@ -1,70 +1,66 @@
...
 
-version = '2.0.4.dev0'
+version = "2.0.4.dev0"
 
 additional_install_requires = []
 
-if sys.platform[:3].lower() == 'win':
-    additional_install_requires += ['nt_svcutils']
+if sys.platform[:3].lower() == "win":
+    additional_install_requires += ["nt_svcutils"]
```
Comment (mauritsvanrees) [inline setup.py:12]: While you are at it, could you remove the `additional_install_requires` and put this directly in `install_requires` please?

```
nt_svcutils; platform_system == 'Windows',
```

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
Mostly good, thanks!  One inline comment with a small request for Windows, but that is really unrelated to this PR.

But with the `plone/meta` changes we also run `pyupgrade` to convert the code to Python 3.8+ only.  And latest `black` is run.  So this will break Python 2 compatibility.  So this is a breaking change.  

I am okay with that.  But it means the setup.py can be cleaned up a bit more, changelog updated. maybe the readme.

I have created a 2.x branch from current master, and have let coredev 5.2 use this.

**Comment by mauritsvanrees** [comment]
BTW, I don't think this has any influence on Jenkins, so those jobs do not need to run.  We should let mr.roboto ignore this repo.

**Comment by mauritsvanrees** [comment]
> BTW, I don't think this has any influence on Jenkins, so those jobs do not need to run. We should let mr.roboto ignore this repo.

Okay, I added this in a `mr.roboto` PR that I already created earlier today for something else.

File: setup.py (hunk 1 above)
Comment (petschki) [reply setup.py:12]: Will fix that too

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
Can anyone activate GHA here?

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py]: Is this dependency still valid? There is only one release from 2010 and the code in  e.g. in https://github.com/zopefoundation/nt_svcutils/blob/d8a19cf70d33d3a6d971f1a8f60b1e261c0caf8f/src/nt_svcutils/service.py#L245 is not Python 3 compatible, it es even a SyntaxError.

Comment (icemac) [reply setup.py]: As there was no interest in this package it got archived in February 2020.

Comment (petschki) [reply setup.py]: OK, then I recommend to drop the dependency here. But I do not have a Windows machine here to test the windows zeoctl services. We should setup a github action with this plattform for the tests.

**Comment by mauritsvanrees** [comment]
> Can anyone activate GHA here?

The central workflow?  Done.

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
@icemac we are pinning `zope.mkzeoinstance=5.0`  for Plone 6.0 because on `5.1` there's  a traceback:

```
Traceback (most recent call last):
  File "/Users/peterm/.pyenv/versions/3.11.3/envs/py-3.11-familienpass/lib/python3.11/site-packages/zc/buildout/buildout.py", line 2252, in main
    getattr(buildout, command)(args)
  File "/Users/peterm/.pyenv/versions/3.11.3/envs/py-3.11-familienpass/lib/python3.11/site-packages/zc/buildout/buildout.py", line 856, in install
    installed_files = self[part]._call(recipe.install)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/peterm/.pyenv/versions/3.11.3/envs/py-3.11-familienpass/lib/python3.11/site-packages/zc/buildout/buildout.py", line 1652, in _call
    return f()
           ^^^
  File "/Users/peterm/workspace/familienpass/src/plone.recipe.zeoserver/src/plone/recipe/zeoserver/recipe.py", line 88, in install
    ZEOInstanceBuilder().create(location, params)
  File "/Users/peterm/workspace/familienpass/eggs/zope.mkzeoinstance-5.1-py3.11.egg/zope/mkzeoinstance/__init__.py", line 188, in create
    makefile(ZEO_CONF_TEMPLATE, home, "etc", "zeo.conf", **params)
  File "/Users/peterm/workspace/familienpass/eggs/zope.mkzeoinstance-5.1-py3.11.egg/zope/mkzeoinstance/__init__.py", line 256, in makefile
    data = template % kwds
           ~~~~~~~~~^~~~~~
KeyError: 'blob_dir'
```

do we have to take care about that in this recipe, or is this a bug in `zope.mkzeoinstance` ?

**Review by jensens — APPROVED** [review]

**Comment by icemac** [comment]
@petschki The problem with `zope.mkzeoinstance` was probably introduced by https://github.com/zopefoundation/zope.mkzeoinstance/pull/16 which added support for `blob_dir`, so it should be fixed over there. (A PR is welcome.)

**Review by mauritsvanrees — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (mauritsvanrees) [reply setup.py]: The nt_svcutils dependency was dropped.

**Comment by mauritsvanrees** [comment]
> I testet this on my Parallels Win11 and it worked. Tests are also fixed. @mauritsvanrees what do you think?

Looking good now.  Thanks!

**Pull request merged**